.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 10
+      - name: Test
```

Comment:
Just noticed you're missing the actual `npm install`.

Reply:
@fearphage `setup-node` already includes `npm` you can review the build log at https://github.com/gliptak/vimium/runs/4276312033?check_suite_focus=true